tests/support/controller.rs

```diff
@@ -245,28 +264,29 @@ fn run(
                 let _ = listening_tx.send(());
             }
 
+            let name = name.clone();
             let serve = hyper::Server::builder(bind.incoming())
                 .http2_only(true)
                 .serve(move || {
-                    let dst_svc = Mutex::new(dst_svc.clone());
+                    let svc = Mutex::new(svc.clone());
                     hyper::service::service_fn(move |req| {
                         let req =
                             req.map(|body| tower_grpc::BoxBody::map_from(PayloadToGrpc(body)));
-                        dst_svc
-                            .lock()
-                            .expect("dst_svc lock")
+                        svc.lock()
+                            .expect("svc lock")
                             .call(req)
                             .map(|res| res.map(GrpcToPayload))
                     })
                 })
-                .map_err(|e| println!("controller error: {:?}", e));
+                .map_err(move |e| println!("{} error: {:?}", name, e));
 
             runtime.spawn(serve);
-            runtime.block_on(rx).expect("support controller run");
+            runtime.block_on(rx).expect(name);
         })
         .unwrap();
 
     listening_rx.wait().expect("listening_rx");
+    println!("{} listening; addr={:?}", name, addr);
```

Comment:
rm or make a proper debug line

Reply:
nevermind, i see that this already does println!s for what i assume are good reasons.

Reply:
@olix0r most of the test support code uses `println!`; the idea is that it should be captured and only printed out if the test fails (if printed from the main test thread). I tried to follow the existing convention with the rest of the test support code. 

Happy to change it though.